### PR TITLE
update maxMdLengthByte to be consistent with validation 

### PR DESCRIPTION
**Pull request by lyndaidaii**
Summary of the changes (in less than 80 characters):

Issue: inconsistent max readme file size between validation during upload and preview readme at manage package page

validation readme max size:
https://github.com/NuGet/NuGetGallery/blob/c71e12429b108c4d87e2da41790b164149a43389/src/NuGetGallery/Services/PackageMetadataValidationService.cs#L61
Preview at manage package: 
https://github.com/NuGet/NuGetGallery/blob/db62de43e24b704e9ab16d8abc39bdfa3e73260b/src/NuGetGallery/Services/ReadMeService.cs#L24

Fix: Make maxmdreadme length to be consistent while uploading and preview

Before:
reamdbug

After: 
bugafter

Addresses https://github.com/NuGet/NuGetGallery/issues/8771

### CODE REVIEW

**Comment by joelverhagen** [comment]
This needs to be rebased on DEV I think. It has some of @dannyjdev's hotfix `main` commits.

File: src/NuGetGallery/Services/ReadMeService.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ internal class ReadMeService : IReadMeService
         internal const string TypeFile = "File";
         internal const string TypeWritten = "Written";
 
-        internal const int MaxMdLengthBytes = 8000;
+        internal const int MaxMdLengthBytes = 1024 * 1024;
```
Comment (joelverhagen) [inline src/NuGetGallery/Services/ReadMeService.cs:24]: Can this use the same constant value as the other 1 MB limit?

**Review by joelverhagen — CHANGES_REQUESTED** [review]
⌚

**Comment by lyndaidaii** [comment]
rebase dev

**Pull request closed without merging**